osquery/sql/sqlite_operations.cpp

```diff
@@ -74,5 +92,8 @@ void registerOperationExtensions(sqlite3* db) {
                           nullptr,
                           addCarveFile,
                           executeCarve);
+
+  sqlite3_create_function(
+      db, "sleep", 1, SQLITE_UTF8, nullptr, sqlSleep, nullptr, nullptr);
```

Comment:
This will be handy outside testing.

One thing I've found, is that working with extensions, I can't do simple single shot command lines -- the extension hasn't always loaded. So being able to shim in a sleep seems nice. (Mostly this is a debugging aid)

Reply:
I don't think this will help in the latent extension loading case. I suspect SQL will still parse your query and early abort if your table does not exist.

Reply:
Ah, yeah. You're probably right. Well... we'll see what proves useful. 

Kinda interesting to think about sample; sleep; sample, and doing some diff calculations in sql. Probably not practical, but who am I to say. I like providing platform tools